src/lpython/parser/parser_stype.h

```diff
@@ -43,6 +43,16 @@ struct Fn_Arg {
     Args_ *args;
 };
 
+struct Kw_or_Star_Arg {
+    LPython::AST::expr_t *star_arg;
+    LPython::AST::keyword_t *kw_arg;
+};
```

Comment:
There was a need for storing `expr_t` and `keyword_t` to be stored under the same type, So I created the above struct to do the same.